src/v/kafka/server/group_metadata_migration.cc

```diff
@@ -622,7 +622,10 @@ ss::future<> group_metadata_migration::migrate_metadata() {
               _controller,
               *group_topic_assignment,
               default_deadline());
-            continue;
+            if (topics.contains(
```

Comment:
How does this change reflect the commit message where it claims a delay is being injected? (in the case the topic doesn't exist after the create attempt)

Reply:
if the topic doesn't exist then right below this we sleep.